source/Features.rst

```diff
@@ -338,20 +328,15 @@ The project status changed to “Stopped”. In a Stopped state all underlying r
 
 If there are any failed provisioned product in my products panel you cannot do actions on the project. You need to terminate that product.
 
-If there are any failed provisioned product in my products panel you cannot do actions on the project. You need to terminate that product.
 
 Budgets (for Principal Investigator)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We have brought in a budget  screen for the Principal Investigator to view budget consumption across projects.
+As a Principal Investigator, you can view the organization-wide budgets from the **Budgets** screen with drill-down to the project, researcher and product level.
 
 **Navigation to Budget screen**
 
-Logged as Principal Account. Click on “☰” Symbol which is available  on left side header. You can see menu like: 
-
-  a. **My projects** : Through this, you can navigate to My Projects page.
-
-  b. **Budget**  : Through this, you can navigate to the Budget Details page. 
+Logged as Principal Account. Click on “☰” Symbol which is available on left side header. Click on the "Budgets" menu item through this, you can navigate to the Budget Details page.  
```

Comment:
Replace "Logged as Principal Account" with "Sign in as the Principal Investigator"